src/pages/DocsPage.tsx

```diff
@@ -99,6 +103,8 @@ function DocsPage() {
               {/*   <span className="switchSun"> <span style={theme === 'light' ? { color: '#ffc045' } : { color: 'grey' }}>☀</span></span>
               <div className="switchMoon"> <span style={theme === 'light' ? { color: 'grey' } : { color: 'white' }}>☾</span></div> */}
               <label className="switch__label">
+                <FontAwesomeIcon icon={faSun} className="faSun" />
+                <FontAwesomeIcon icon={faMoon} className="faMoon" />
```

Comment:
It looks like the chechbox is just covering one of the icons each time. You can see the sun icon peeking out of the checkbox: 
![image](https://user-images.githubusercontent.com/59885733/150620589-e2c990da-378e-4568-ab10-c2d7a873349e.png)

Let's these icons render conditionally depending on the theme, like this:
` <FontAwesomeIcon> { theme === "light" ? icon={faMoon} className="faMoon" : icon={faSun} className="faSun" } />`